src/styles/utilities/_jca-keyword.scss

```diff
@@ -0,0 +1,8 @@
+.jca-keyword {
+  display: inline-flex;
+  padding: 3px 10px;
+  border-radius: $default-radius;
+  background: var(--info);
+  color: var(--primary);
+  font-family: 'Oxygen Mono', monospace;
+}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/manual/hello_100/jca-ui-kit/pullRequest?prid=5693353)